### DISCUSSION TITLE
正版皮肤在游戏内无法显示

### DISCUSSION THREAD

**Discussion by PolarisG7** [Meloong-Git/PCL#5191] [提问]
[之前发起过提问，因为忙没留意邮箱，大致情况是启动器页面正版头像正常显示，游戏内无法显示，尝试过多种方法未能解决，以下是启动器的log和游戏logs文件里面的log
[log.zip](https://github.com/user-attachments/files/17944591/log.zip)

**Comment by zkitefly** [comment]
有试过使用其他启动器吗？

**Reply by PolarisG7** [reply]
HMCL启动器和官方启动器是正常的，我用HMCL启动器打开PCL目录下的游戏，他会提示游戏环境不支持，然后给我重新下载还是什么的，进去之后就是正常显示皮肤的了

**Reply by 3gf8jv4dv** [reply]
> 我用HMCL启动器打开PCL目录下的游戏，他会提示游戏环境不支持，然后给我重新下载还是什么的，进去之后就是正常显示皮肤的了

能否提供下截图和相应的步骤？我看了你的日志，感觉像是网络问题，但你这么说又感觉不对劲。

**Reply by PolarisG7** [reply]
很抱歉，那个界面弹出了一次之后就没有出现过了，之后我打开其他游戏版本也没有提示了

**Reply by PolarisG7** [reply]
> > 我用HMCL启动器打开PCL目录下的游戏，他会提示游戏环境不支持，然后给我重新下载还是什么的，进去之后就是正常显示皮肤的了
> 
> 能否提供下截图和相应的步骤？我看了你的日志，感觉像是网络问题，但你这么说又感觉不对劲。

我更换了游戏java，将原来自带的jdk21更换成我重新下载的jdk21就正常了，以及我之前说的弹窗又出现了一次，也是我会去更换游戏java的依据
<img width="937" alt="屏幕截图 2024-11-29 121923" src="https://github.com/user-attachments/assets/e90eacaa-1a1d-426b-af3f-8ffda578b791">
<img width="524" alt="屏幕截图 2024-11-29 160002" src="https://github.com/user-attachments/assets/246a301a-0387-4949-b6e9-87ac596f9e20">

**Comment by AnemoFlower** [comment]
因网络问题无法连接 Mojang 身份验证服务器（连接超时）导致的无法获取皮肤数据，安装Mod [CustomSkinLoader](https://modrinth.com/mod/customskinloader) 或许可以解决这个问题。

**Reply by allMagicNB** [reply]
本质上来说没用。
